src/goto-harness/goto_harness_main.cpp

```diff
@@ -10,6 +10,6 @@ Author: Diffblue Ltd.
 
 int main(int argc, const char *argv[])
 {
-  auto parse_options = goto_harness_parse_optionst(argc, argv);
+  goto_harness_parse_optionst parse_options{argc, argv};
   return parse_options.main();
```

Comment:
While at it, go one step further and do `return goto_harness_parse_optionst{argc, argv}.main();`?